etcdserver/api/v3rpc/maintenance.go

```diff
@@ -19,12 +19,17 @@ import (
 
 	"github.com/coreos/etcd/etcdserver"
 	pb "github.com/coreos/etcd/etcdserver/etcdserverpb"
+	"github.com/coreos/etcd/mvcc"
 	"github.com/coreos/etcd/mvcc/backend"
 	"github.com/coreos/etcd/pkg/types"
 	"github.com/coreos/etcd/version"
 	"golang.org/x/net/context"
 )
 
+type KVGetter interface {
+	KV() mvcc.ConsistentWatchableKV
```

Comment:
@xiang90 it complains like `*etcdserver.EtcdServer does not implement KVGetter (wrong type for KV method)` because `EtcdServer` only implements `KV() mvcc.ConsistentWatchableKV` in `etcdserver` pkg.